index.html

```diff
@@ -17,75 +17,89 @@
             some of my work and experience! If you like what you see and have a
             project you need coded, don’t hestiate to contact me.
           </p>
-          <ul class="icons">
-            <li>
-              <a href="#"> <i class="fa-brands fa-github"></i></a>
-            </li>
-            <li>
-              <a href="#"> <i class="fa-brands fa-linkedin-in"></i></a>
-            </li>
-            <li>
-              <a href="#"> <i class="fa-brands fa-angellist"></i></a>
-            </li>
-            <li>
-              <a href="#"> <i class="fa-brands fa-facebook"></i></a>
-            </li>
-            <li>
-              <a href="#"> <i class="fa-brands fa-twitter"></i></a>
-            </li>
-          </ul>
-        </section>
-      </header>
+        </div>
+
+        <ul class="icons">
+          <li>
+            <a href="#"> <i class="fa-brands fa-github"></i></a>
+          </li>
+          <li>
+            <a href="#"> <i class="fa-brands fa-linkedin-in"></i></a>
+          </li>
+          <li>
+            <a href="#"> <i class="fa-brands fa-angellist"></i></a>
+          </li>
+          <li>
+            <a href="#"> <i class="fa-brands fa-facebook"></i></a>
+          </li>
+          <li>
+            <a href="#"> <i class="fa-brands fa-twitter"></i></a>
+          </li>
+        </ul>
+      </section>
 
       <!-- Work section -->
       <section class="work-section">
-        <h1 class="work-title">My Recent Works</h1>
-        <hr />
-        <img src="img/Placeholder.png" alt="Placeholder" />
-        <h2 class="work-subtitle">Multi-Post Stories</h2>
-        <p class="work-desc">
-          A daily selection of privately personalized reads; no accounts or
-          sign-ups required. has been the industry's standard dummy text ever
-          since the 1500s, when an unknown printer took a standard dummy text.
-        </p>
-        <ul class="work-section-list">
-          <li><a href="#">css</a></li>
-          <li><a href="#">html</a></li>
-          <li><a href="#">bootstrap</a></li>
-          <li><a href="#">Ruby</a></li>
-        </ul>
-        <a href="#" class="project-btn">See project</a>
+        <div class="work-title-container">
+          <h1 class="work-title">My Recent Works</h1>
+          <hr />
+        </div>
+        <div class="work-flex-container">
+          <img
+            src="img/plcholder.png"
+            alt="Placeholder"
+          />
+          <div class="work-flex-desc">
+            <h2 class="work-subtitle">Multi-Post Stories</h2>
+            <p class="work-desc">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard dummy text
+              ever since the 1500s, when an unknown printer took a standard
+              dummy text.
+            </p>
+            <ul class="work-section-list">
```

Comment:
I dont know why are your seeing that in your screen because in my screen it is working just fine..
![Screenshot](https://user-images.githubusercontent.com/96125730/188123106-9a5fd3a6-2bc2-4fb2-b610-3965a91c1e03.png)